TrainingProject/src/main/java/com/example/trainningspringproject/controller/MachineController.java

```diff
@@ -0,0 +1,62 @@
+package com.example.trainningspringproject.controller;
+
+import com.example.trainningspringproject.entity.Machine;
+import com.example.trainningspringproject.service.MachineService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.List;
+import java.util.Optional;
+
+@Controller
+public class MachineController {
+
+    @Autowired
+    private MachineService service;
+
+    @GetMapping("/")
+    public String viewHomePage(Model model) {
```

Comment:
change the name of the API method to be getAllMachines
and the method should take Machine obj